### ISSUE TITLE
stash message is prefixed with --

### ISSUE THREAD

**Issue by carlolars** [ForkIssues/TrackerWin#502]
When saving a stash with a *stash message* the message is prefixed with `--`.

Fork use the command `git stash save` which should not have the double-dashes:
```
$ git stash save -- My message
Saved working directory and index state On master: -- My message
# should be:
$ git stash save My message
Saved working directory and index state On master: My message
```

**Comment by DanPristupov** [comment]
Fixed in 1.41. This problem was causes by a change in git behavior.

**Comment by carlolars** [comment]
Working as expected, thanks!
